### DISCUSSION TITLE
Mute rules: are disabled rules cleaned up automatically?

### DISCUSSION THREAD

**Discussion by jayluo667** [ccfos/nightingale#1307] [Q&A]
前端版本：5.14.2   后端版本：v5.14.2-f8ac0a9b4ad9657b71b34a200b42a80e474d71d6
问题描述：屏蔽规则-未启用的规则会被自动清理？是特地设置的还是bug？

**Comment by UlricQin** [comment]
过期的屏蔽规则会被自动清理
